tests/legacy-cli/e2e/assets/1.0-project/src/app/app.module.ts

```diff
@@ -1,7 +1,6 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
-import { HttpModule } from '@angular/http';
```

Comment:
`@angular/http` wasn't actually used in this app and is deprecated, causing problems after updating in the test.